### ISSUE TITLE
Number Highlighting broken

### ISSUE THREAD

**Issue by danjenson** [neoclide/coc.nvim#1699]
Somewhere between this commit:

https://github.com/neoclide/coc.nvim/commit/88f5eeb613f6c780de715fe721e3d41814cf2cc3

And the most recent commit on `release` enableHighlightLineNumber has broken. Line numbers associated with errors/warnings/etc are no longer changing color in nvim (v0.4.3).  Here are the signs that are set, even with `diagnostic.enableHighlightLineNumber` and `diagnostic.enableSign` explicitly set to true in coc-settings.json:

```
sign CocError text=>> linehl=CocErrorLine texthl=CocErrorSign
sign CocWarning text=⚠  linehl=CocWarningLine texthl=CocWarningSign        
sign CocInfo text=>> linehl=CocInfoLine texthl=CocInfoSign
sign CocHint text=>> linehl=CocHintLine texthl=CocHintSign
sign CocSelected text=*  linehl=CocSelectedLine texthl=CocSelectedText
```

**Comment by chemzqm** [comment]
Upgrade your coc.nvim.

**Comment by danjenson** [comment]
Why, it was released in version 0.3.2?

https://github.com/neovim/neovim/releases/tag/v0.3.2

**Comment by alex-popov-tech** [comment]
@chemzqm hello, should it work now ( with 78 release ) ? cause it seems to be still broken :(
